PIL/ImageShow.py

```diff
@@ -156,12 +156,16 @@ def show_file(self, file, **options):
 
     # implementations
 
+    # The default Image Viewer of Ubuntu using the command "eog" in the terminal.
+    # Reference: https://help.gnome.org/users/eog/stable/commandline.html.en
+
     class DisplayViewer(UnixViewer):
         def get_command_ex(self, file, **options):
-            command = executable = "display"
+            command = executable = "display" or "eog"
```

Comment:
Doesn't this just evaluate to "display"?

Reply:
yap just evaluate to display  :smiley:

Reply:
I understand that, but what's the point of changing this line? `command == display` with or without this change.

Reply:
If I got you, I keep display for the other *nix user.

Reply:
On every Unix, `command` will always be `"display"`. It will never be `"eog"`. 

Have you tested this PR? Does it open eog?

Reply:
Yah I test it on my Ubuntu machine, If I add ```eog``` only that time ```PIL.Image.Image.show``` work otherwise ```PIL.Image.Image.show``` don't work :)

Reply:
Maybe a new class EogViewer.

```python
    class EogViewer(UnixViewer):
        def get_command_ex(self, file, **options):
            command = executable = "eog"
            return command, executable
```
And then:
```python
    if which("display"):
        register(DisplayViewer)
    elif which("eog"):
        register(EogViewer)
```

Please test it works.

Reply:
@hugovk  thanks but coincidentally I was writing the same code after your comment ```command = executable = "display" or "eog"```